traitsui/testing/tester/wx/helpers.py

```diff
@@ -135,6 +227,43 @@ def key_click_text_ctrl(control, interaction, delay):
         control.WriteText(interaction.key)
 
 
+def key_click_combobox(control, interaction, delay):
+    """ Performs simulated typing of a key on the given wxComboBox
+    after a delay.
+
+    Parameters
+    ----------
+    control : wxComboBox
+        The wx Object to be acted on.
+    interaction : instance of command.KeyClick
+        The interaction object holding the key input
+        to be simulated being typed
+    delay : int
+        Time delay (in ms) in which the key click will be performed.
+    """
+    if not control.IsEditable():
+        raise Disabled("{!r} is disabled.".format(control))
```

Comment:
This does not work for combobox when the UI is disabled...
Looks like we should do this instead:
```
if not (control.IsEnabled() and control.IsEditable()):
```

This is the test I expect to pass but currently fails on wx (passes on Qt though):
```

    def test_simple_editor_disabled(self):
        enum_edit = EnumModel(value="two")
        view = View(
            UItem(
                "value",
                style="simple",
                enabled_when="value == 'one'",
                editor=EnumEditor(evaluate=True, values=["one", "two"]),
            ),
        )
        tester = UITester()
        with tester.create_ui(enum_edit, dict(view=view)) as ui:
            combobox = tester.find_by_name(ui, "value")
            with self.assertRaises(Disabled):
                combobox.perform(command.KeyClick("Enter"))

```

(this applied to KeySequence as well)